src/main/java/com/microsoft/aad/msal4j/MSALRuntimeAuthResult.java

```diff
@@ -0,0 +1,112 @@
+package com.microsoft.aad.msal4j;//TODO: will be move to JavaMSALRuntime package, must not reference MSAL Java directly
+
+import com.sun.jna.Structure;
+import com.sun.jna.WString;
+import com.sun.jna.ptr.IntByReference;
+import com.sun.jna.ptr.LongByReference;
+
+import java.util.concurrent.CompletableFuture;
+
+//FieldOrder needs to explicitly state the public fields that should be in the C structure JNA creates
+@Structure.FieldOrder({"authParamsHandle", "accountHandle", "authResultHandle", "idToken", "accessToken", "accessTokenExpirationTime", "accountId", "accountClientInfo"})
+public class MSALRuntimeAuthResult extends Structure implements MSALRuntimeLibrary.AuthResultCallbackData {
+    public LongByReference authParamsHandle;
+    public LongByReference accountHandle;
+    public LongByReference authResultHandle;
+    public MSALRuntimeUtils.StringByReference idToken = new MSALRuntimeUtils.StringByReference(new WString(""));
+    public MSALRuntimeUtils.StringByReference accessToken = new MSALRuntimeUtils.StringByReference(new WString(""));
+    public LongByReference accessTokenExpirationTime;
+    public MSALRuntimeUtils.StringByReference accountId = new MSALRuntimeUtils.StringByReference(new WString(""));//TODO: split AuthenticationResult into SignInResult (id token/account) and TokenResult (access tokens)? Or keep it simple all-in-one like MSAL Java's AuthenticationResult?
+    public MSALRuntimeUtils.StringByReference accountClientInfo = new MSALRuntimeUtils.StringByReference(new WString(""));
+    private MSALRuntimeLibrary MSALRuntime_INSTANCE;
+    private MSALRuntimeError MSALRuntimeErrorHandler;
+
+    public void setMSALRuntimeLibraryInstance (MSALRuntimeLibrary MSALRuntime_INSTANCE) {
+        this.MSALRuntime_INSTANCE = MSALRuntime_INSTANCE;
+    }
+
+    public void setMSALRuntimeErrorHandler (MSALRuntimeLibrary MSALRuntime_INSTANCE) {
+        this.MSALRuntimeErrorHandler = new MSALRuntimeError();
+        this.MSALRuntimeErrorHandler.MSALRuntime_INSTANCE = MSALRuntime_INSTANCE;
+    }
```

Comment:
This method doesn't do what the name implies?

Reply:
It did in an earlier version, but I missed renaming it to something like 'initializeErrorHandler'. Regardless, this method has  been removed entirely as the error handler class has been refactored to just a helper class with static methods.